pkg/template/apis/template/validation/validation.go

```diff
@@ -79,6 +80,17 @@ func ValidateTemplateInstance(templateInstance *templateapi.TemplateInstance) (a
 		for _, msg := range oapi.GetNameValidationFunc(uservalidation.ValidateUserName)(templateInstance.Spec.Requester.Username, false) {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("spec.requester.username"), templateInstance.Spec.Requester.Username, msg))
 		}
+		if templateInstance.Spec.Requester.Groups != nil {
+			// unlike users, the system groups may have colons in them, starting with "system:", and will be automatically appended to users group lists
+			for _, group := range templateInstance.Spec.Requester.Groups {
+				if strings.HasPrefix(group, "system:") {
+					continue
+				}
+				for idx, msg := range oapi.GetNameValidationFunc(uservalidation.ValidateGroupName)(group, false) {
```

Comment:
> @deads2k on what basis do Validate{User,Group}Name reject names containing :? Is there a mechanism by which names containing : may be accepted?

Not for users today.  I don't think we've settled how we're going to reconcile with upstream on this.  @liggitt @openshift/sig-security